### PR TITLE
April 2023 Update

### PR DESCRIPTION
**Pull request by dholbach**

### CODE REVIEW

File: content/en/blog/2023-05-01-april-update/index.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+### Your Community Team
+
+### People writing/talking about Flux
+
+We love it when you all write about Flux and share your experience,
+write how-tos on integrating Flux with other pieces of software or other
+things. Give us a shout-out and we will link it from this section! ✍
+
+- <https://grafana-operator.github.io/grafana-operator/blog/2023/03/29/install-grafana-operator-using-flux-and-kustomize/>
```
Comment (NissesSenap) [inline content/en/blog/2023-05-01-april-update/index.md:127]: For some additional context around the grafana-operator link.
How about something like this?

The grafana-operator team have recently started to ship their Kustomize manifests using OCI with the help of Flux artifact. As a part of this, they have written a small blog on how to install grafana-operator using Flux and how to manage grafana dashboards as code.

File: content/en/blog/2023-05-01-april-update/index.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+> We're now working together to expand the Bug Scrub, the first AEST "bug scrub"
+> session was last week. This is meant to include more time zones who would
+> typically not be able to attend (AEST-centric scheduling would be more accessible
+> to Eastern Europe, India, Southeast Asia, etc.)
+
+In addition to that Michael was also involved through being a co-founder at
+[k8s@Home](https://github.com/k8s-at-home/)
+Github & [Discord](https://discord.gg/sTMX7Vh) where they support Flux users as
+well.
+
```
Comment (dholbach) [inline content/en/blog/2023-05-01-april-update/index.md:136]: @xunholy Does this look good to you? I could also include https://avatars.githubusercontent.com/u/20387402 as your profile pic if you like?

**Comment by dholbach** [comment]
→ https://deploy-preview-1447--fluxcd.netlify.app/blog/

File: content/en/blog/2023-05-01-april-update/index.md [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Weave GitOps
+
```
Comment (dholbach) [inline content/en/blog/2023-05-01-april-update/index.md:61]: @jamwils Anything you'd like to add here for April?

File: content/en/blog/2023-05-01-april-update/index.md [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.22.0>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.21.2>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.21.1>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.21.0>
+
+#### Terraform-controller
+
```
Comment (dholbach) [inline content/en/blog/2023-05-01-april-update/index.md:68]: @chanwit  Anything you'd like to add here for April?

File: content/en/blog/2023-05-01-april-update/index.md [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+#### Flux Subsystem for Argo
+
+- <https://www.weave.works/blog/flamingo-expand-argo-cd-with-flux>
+
+#### Azure GitOps
+
+#### VS Code GitOps Extension
+
```
Comment (dholbach) [inline content/en/blog/2023-05-01-april-update/index.md:76]: @juozasg Anything you'd like to add here for April?

File: content/en/blog/2023-05-01-april-update/index.md [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+sure to mention it in the next post!
+
+### Upcoming Events 📆
+
+We are happy to announce that we have a number of events coming up in
+May - tune in to learn more about Flux and GitOps best practices,
+get to know the team and join our community.
+
+#### Flux Bug Scrub
+
```
Comment (dholbach) [inline content/en/blog/2023-05-01-april-update/index.md:102]: @kingdonb  Anything you'd like to add here for April?

**Comment by dholbach** [comment]
Does anyone want to check if there's entries about Flux in https://www.youtube.com/@cncf/videos or at https://www.flickr.com/photos/143247548@N03/?

**Comment by dholbach** [comment]
@staceypotter found all of these:

https://www.flickr.com/photos/143247548@N03/52837739714/in/album-72177720307596641/
https://www.flickr.com/photos/143247548@N03/52836983222/in/album-72177720307596641/
https://www.flickr.com/photos/143247548@N03/52837980010/in/album-72177720307596641/
https://www.flickr.com/photos/143247548@N03/52837980005/in/album-72177720307596641/
https://www.flickr.com/photos/143247548@N03/52837568501/in/album-72177720307596641/
https://www.flickr.com/photos/143247548@N03/52838225580/in/album-72177720307596641/
https://www.flickr.com/photos/143247548@N03/52838007744/in/album-72177720307596641/
https://www.flickr.com/photos/143247548@N03/52838267158/in/album-72177720307596641/
https://www.flickr.com/photos/143247548@N03/52838006324/in/album-72177720307596641/
https://www.flickr.com/photos/143247548@N03/52837250597/in/album-72177720307596641/
https://www.flickr.com/photos/143247548@N03/52838267043/in/album-72177720307596641/
https://www.flickr.com/photos/143247548@N03/52838267018/in/album-72177720307596641/
https://www.flickr.com/photos/143247548@N03/52838969604/in/album-72177720307596641/

File: content/en/blog/2023-05-01-april-update/index.md (hunk 3 above)
Comment (makkes) [reply content/en/blog/2023-05-01-april-update/index.md]: https://www.weave.works/blog/weave-gitops-release-2023-04 maybe?

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @dholbach 🥇

**Pull request merged**